### DISCUSSION TITLE
Basic usage question: action fails with Input required and not supplied: upload_url

### DISCUSSION THREAD

**Discussion by nofishonfriday** [ncipollo/release-action#232] [Q&A] [answered]
Hi,

I'm pretty new to GitHub actions so a little guidance would be appreciated.

My workflow yml (linked where I'm using your action):
https://github.com/nofishonfriday/pamplejuce/blob/main/.github/workflows/cmake_ctest.yml#L126

As can be seen in the [action log](https://github.com/nofishonfriday/pamplejuce/actions/runs/2605750900) the artifacts are there, but the action fails with following output:
`Input required and not supplied: upload_url`

Also, why does it fail finding the artifacts although they seem to be present.
edit: Also asked [here](https://github.com/ncipollo/release-action/discussions/127), but not solved.

Screenshot of the action log:
![action log](https://user-images.githubusercontent.com/7658194/177050412-5a811b08-ad8b-4c27-85e7-cfbd3fe3e865.jpg)

Thank you.

**Comment by ncipollo** [comment, marked as answer]
>  Input required and not supplied: upload_url

This isn't coming from the `release_action` . This is coming from your `Upload Linux Asset` step, which uses an entirely different action. It looks like it's trying to reference the output of `create_release`, which you've commented out.

> Also, why does it fail finding the artifacts although they seem to be present

The error is indicated that the glob you are providing isn't matching any files within the working directly. It might be that they are named differently then you think, or they are in some sub-directly, not within your working directory. To solve issues like this, I usually throw in a shell script step and drop a few `ls` command to see what's in the working directory, and other major directories. My best guess is your `download-artifact` is either not downloading anything or it's downloading the results to a subfolder.
